reference/unity_to_godot.rst

```diff
@@ -0,0 +1,177 @@
+|                   | | Large assets store                                                              | | Signals                                                                                                      |
+|                   |                                                                                   | | Animation tool                                                                                               |
+|                   |                                                                                   | | Shader graphs                                                                                                |
+|                   |                                                                                   | | GDScript                                                                                                     |
+|                   |                                                                                   | | Integrated debugger                                                                                          |
++-------------------+-----------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+
+
+
+The editor
+----------
+
+Godot Engine provides a rich-featured editor that allows you to build your games. The pictures below display both editors with colored blocks to indicate common functionalities.
+
+.. image:: /img/unity-gui-overlay.png
+.. image:: /img/godot-gui-overlay.png
+
+
+Note that Godot editor allows you to dock each panel at the side of the scene editor you wish.
+
+As you can see, there are some similarities between both editors: first of course, the scene which allows you to display and organize your game elements on screen. Second, the inspector which allows you to edit the parameters of the selected object. Also, the Hierarchy panel in Unity finds its correspondance with Godot's Scene tree panel, although we will see in the next section the differences between how Unity and Godot manage scenes and scene elements.
+
+Unity and Godot both directly use the folder architecture you define for your project. There is no real good solution, and every folder architecture will work for both engines. You can thus continue with your current way to organize your project. However, we have some advices to share a bit further, in order to match your project architecture and Godot scene system.
+
+To finish, the toolbar at the top of the screen is also similar between Unity and Godot. However, Godot provides some new buttons aside the classic "Play", "Pause" and "Stop" buttons. These buttons are introduced in the `Toolbar <introduction/editor#toolbar>`_ section as they a bit out of the scope of this page.
+
```

Comment:
I would use a simpler and more to the point language, and trying to remark what Godot advantages are, like:

While both may seem similar, there are many differences below the surface. Both let you organize the project using the filesystem, but Godot approach is simpler, with a single configuration file, minimalist text format, and no metadata. All this contributes to Godot being much friendlier to VCS systems such as Git, Subversion or Mercurial.

The scene panel is similar to the hierarchy but, as each node has a specific function, the approach used by Godot is more visually descriptive. In other words, It's easier to guess how a scene works at a glance.

The inspector in Godot is more minimalist and designed to only show properties. Thanks to this, objects can export a much larger amount of useful parameters to the user, without having to hide functionality in language APIs. As a plus, Godot allows animating any of those properties visually, so changing colors, textures, enumerations or even links to resources in real-time is possible without involving code.

Finally, the toolbar at the top of the screen is similar in the sense that it allows controlling the project playback, but projects in Godot run in a separate window, as they don't execute inside the editor (but the tree and objects can still be explored in the debugger window). This approach has the disadvantage that the running game can't be explored from different ways (though this may be supported in the future, and displaying collision gizmos in the running game is already possible), but in exchange it has several advantages:

- Running the project and closing it is very fast (Unity has to save, run the project, close the project and then reload the previous state).
- Live editing is a lot more useful, because changes done to the editor take effect immediately in the game, and are not lost (nor have to be synced) when the game is closed. This allows fantastic workflows, like creating levels while you play them. You can even program while the game is running and scripts are reloaded automatically.
-The editor is more stable, because the game runs in a separate process

Finally, the top toolbar includes a menu for remote debugging. These options make it very easy to deploy to a device (connected phone, tablet or browser via HTML5), and debug / live edit the running game after it was exported.